src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinClientPlayerEntity.java

```diff
@@ -189,4 +193,16 @@ private void injectForcedState(CallbackInfoReturnable<Boolean> cir) {
         }
     }
 
+    @Inject(method = "swingHand", at = @At("HEAD"), cancellable = true)
+    private void swingHand(Hand hand, CallbackInfo callbackInfo) {
+        if (ModuleNoSwing.INSTANCE.getEnabled()) {
+            if (ModuleNoSwing.INSTANCE.getEnabled()) {
```

Comment:
Checking if the module is enabled two times.

Reply:
holy moly did not see that